From 9c46e69556ac58dda20d7bbfc88fe5fdc65be2ed Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Tue, 14 May 2019 13:23:06 -0400
Subject: [PATCH] Clean up poweredby images

Forcing the sizes results in terrible output. Also, we should always
display just the Fedora icon and whichever webserver is in use. The
httpd and nginx packages will be updates to ensure that
/poweredby.png is appropriate.

Signed-off-by: Stephen Gallagher <sgallagh@redhat.com>
---
 testpage/index.html | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/testpage/index.html b/testpage/index.html
index 8583689d4b7cf7431e496c4ec52bef3d5e61071d..555e9fd54d9ef9ebb0b6aa193c5d29e0ec62a91f 100644
--- a/testpage/index.html
+++ b/testpage/index.html
@@ -160,13 +160,12 @@
         in the <strong>nginx</strong> configuration file
         <code>/etc/nginx/nginx.conf</code>.</p>
       </div>
 
       <div class="logos">
-        <a href="https://getfedora.org/"><img src= "/icons/poweredby.png" alt="[ Powered by Fedora ]" width= "88" height="31" /></a> <!-- apache -->
-        &nbsp;
-        <a href="https://getfedora.org/"><img src="poweredby.png" alt="[ Powered by Fedora ]" width="88" height="31" /></a> <!-- nginx -->
+        <a href="https://getfedora.org/"><img src= "/icons/poweredby.png" alt="[ Powered by Fedora ]" /></a> <!-- Fedora -->
+        <img src="poweredby.png" /></a> <!-- webserver -->
       </div>
     </div>
   </div>
 </body>
 </html>
-- 
2.21.0

